modules/tf_cloudbuild_builder/variables.tf

```diff
@@ -70,7 +70,6 @@ variable "gar_repo_name" {
 variable "gar_repo_location" {
   description = "Name of the location for the Google Artifact Repository."
   type        = string
-  default     = "us"
```

Comment:
Do we need to remove the default?  Looking if we can make this no-longer a breaking change.

Reply:
We can remove, let's just document in an upgrade guide similar to https://github.com/terraform-google-modules/terraform-google-bootstrap/blob/master/docs/upgrading_to_v5.0.md (next version will be v9.0)

Reply:
fixed